### ISSUE TITLE
Separate "Scratch" Directories for Different Purposes

### ISSUE THREAD

**Issue by crantila** [cuthbertLab/music21#29]
TL;DR: I'd like to be able to choose different locations for the different types of files stored in the music21 "scratch" directory.

The "scratch" directory is used for two very different things:

* Creating temporary files while preparing output.
* Storing pickled versions of previously-imported output.

The first of these is the definition of "temporary" files, and they could be removed virtually right away after they're created. Certainly they *should* be removed periodically.

The second of these is not at all temporary, since it's only useful if the files are *not* removed periodically (certainly at much longer periods than the temporary files).

Because music21 puts these in the same place, it's more difficult to manage the different roles these files play. For a desktop user, it's not a big deal. For deployment on a server, I want to put some of these files in ``/tmp`` and some of them in ``/var/cache``, and they require different (SELinux) security labels.

For these reasons, I'd like to be able to set different directories for the different types of files.

(Note this is indeed low priority).

**Comment by mscuthbert** [comment]
I'm following the Enhancement model suggested at http://brianegranger.com/?p=249 -- closing the issue here; if others believe it is important, they can reopen the issue.

**Comment by mscuthbert** [comment]
Newest git release allows storing the .json file for a LocalCorpus elsewhere, where the amount of time to recreate is substantial.
